apps/concept-catalog/app/page.tsx

```diff
@@ -1,7 +1,21 @@
-import { redirect } from 'next/navigation';
+import { authOptions, getOrganizationNumber } from '@catalog-frontend/utils';
+import { getServerSession } from 'next-auth';
+import { RedirectType, redirect } from 'next/navigation';
 
 const Home = async () => {
-  redirect(`${process.env.FDK_REGISTRATION_BASE_URI}`);
+  const session = await getServerSession(authOptions);
+
+  if (!session || !(session?.user && Date.now() < session?.accessTokenExpiresAt * 1000)) {
+    redirect(`/auth/signin`);
+  }
+
+  const orgNumber = getOrganizationNumber(`${session?.accessToken}`);
```

Comment:
Hva skjer for folk som har tilgang til flere organisasjoner?
Hva med sys-admin?